ranvier

```diff
@@ -19,9 +19,9 @@ const net = require('net');
 const commander = require('commander');
 const wrap = require('wrap-ansi');
 const argv = require('optimist').argv;
-const longjohn = require('longjohn');
-const pe = require('pretty-error').start();
-pe.skipNodeFiles(); // Ignore native node files in stacktrace.
+//const longjohn = require('longjohn');
+//onst pe = require('pretty-error').start();
```

Comment:
This was accidental, these two break stack traces when using a real debugger so I disable them when deving